### PR TITLE
Add use_online_imu_extrinsics_in_3d parameter and set it to true by default.

### PR DESCRIPTION
**Pull request by CodeArno**
Add the parameter `OptimizationProblemOptions.use_online_imu_extrinsincs_in_3d`. Set it to true by default to not change existing behavior.

Using online IMU extrinsics is not always desirable, and particularly during localization, we have determined that it is preferable to turn this feature off.

### CODE REVIEW

**Comment by CodeArno** [comment]
@wohe PTAL

File: cartographer/mapping/proto/pose_graph/optimization_problem_options.proto [hunk 1]
```diff
@@ -54,6 +54,9 @@ message OptimizationProblemOptions {
   // 3D only: fix Z.
   bool fix_z_in_3d = 13;
 
+  // 3D only: activate online IMU extrinsics.
+  bool online_imu_extrinsics_in_3d = 18;
```
Comment (MichaelGrupp) [inline cartographer/mapping/proto/pose_graph/optimization_problem_options.proto]: `use_online_imu_extrinsics_in_3d`?

Comment (CodeArno) [reply cartographer/mapping/proto/pose_graph/optimization_problem_options.proto]: Done.

**Comment by wohe** [comment]
@CodeArno Seems you have to fix the unit tests that contain optimization problem options.

**Comment by CodeArno** [comment]
@wohe Done, sorry about that.

**Comment by wohe** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user wohe. Merge queue now has a length of 1.

**Pull request merged**